### ISSUE TITLE
ShareButtons does not render out "aria-label" on each button for accessibility 

### ISSUE THREAD

**Issue by dongmei-cao** [MurhafSousli/ngx-sharebuttons#277]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?

Happen on all OS/Version

### Versions

Angular CLI: 1.6.5
Node: 8.9.4
OS: darwin x64
Angular: 
...

### Repro steps

From the latest code

### Desired functionality

For accessibility, the `<share-button>` should have the `aria-label` set for each media. 

The following is a sample HTML for the Twitter `ShareButton` in `ShareButtons`. Notice that the `<svg aria-hidden="true">`. For screen readers or similar technologies, an `aria-label` is needed for the `<share-button>`.

```
<share-buttons >
  <div class="sb-group sb-default">
    <share-button class="sb-button sb-default">
      <button class="sb-wrapper sb-twitter sb-show-icon" style="--button-color:#00acee; font-size: 12.6px;">
        <div class="sb-inner">
          <div class="sb-content">
            <div class="sb-icon">
              <fa-icon class="ng-fa-icon">
                <svg aria-hidden="true" data-prefix="fab" data-icon="twitter" class="svg-inline--fa fa-twitter fa-w-16"
                     role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
                  <path fill="currentColor"
                        d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"></path>
                </svg>
              </fa-icon>
            </div></div>
      </button>
    </share-button>
    <div class="sb-button sb-default"></div>
  </div>
</share-buttons>
```

**Comment by dongmei-cao** [comment]
@MurhafSousli should be the media names. For example, "Twitter", "Facebook", "Email", "LinkedIn", etc

**Comment by dongmei-cao** [comment]
@MurhafSousli Do you think you will make this enhancement and 278 soon? Thanks for the great work!

**Comment by MurhafSousli** [comment]
Added in v6.0.0, `aria-label` will be same as button text, if you wish to change that, you can set it for each button like:

```ts
ShareModule.forRoot({
   prop: {
      facebook: {
         ariaLabel: 'Share on Facebook'
      }
   }
})
```
